huaweicloud/services/ces/resource_huaweicloud_ces_agent_maintenance_task.go

```diff
@@ -0,0 +1,353 @@
+			},
+			"version": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `Specifies the version number.`,
+			},
+			"enable_force_new": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice([]string{"true", "false"}, false),
+				Description:  utils.SchemaDesc("", utils.SchemaDescInput{Internal: true}),
+			},
+			"invocations": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: `The result of the agent maintenance task.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"invocation_id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The task ID.`,
+						},
+						"instance_id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The server ID.`,
+						},
+						"instance_name": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The server name`,
+						},
+						"instance_type": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The server type.`,
+						},
+						"intranet_ips": {
+							Type:        schema.TypeList,
+							Elem:        &schema.Schema{Type: schema.TypeString},
+							Computed:    true,
+							Description: `The private IP address list.`,
+						},
+						"elastic_ips": {
+							Type:        schema.TypeList,
+							Elem:        &schema.Schema{Type: schema.TypeString},
+							Computed:    true,
+							Description: `The EIP list.`,
+						},
+						"invocation_type": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The task type.`,
+						},
+						"invocation_status": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The task status.`,
+						},
+						"invocation_target": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The task object.`,
+						},
+						"create_time": {
+							Type:        schema.TypeInt,
+							Computed:    true,
+							Description: `When the task was created.`,
+						},
+						"update_time": {
+							Type:        schema.TypeInt,
+							Computed:    true,
+							Description: `When the task was updated.`,
+						},
+						"current_version": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The current version of the agent.`,
+						},
+						"target_version": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The target version.`,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func resourceAgentMaintenanceTaskCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	conf := meta.(*config.Config)
+	region := conf.GetRegion(d)
+
+	var (
+		createAgentMaintenanceTaskCreateHttpUrl = "v3/{project_id}/agent-invocations/batch-create"
```

Comment:
```suggestion
		createAgentMaintenanceTaskHttpUrl = "v3/{project_id}/agent-invocations/batch-create"
```
please also  check other variable names